packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -176,7 +176,20 @@ export type CallAdapterClientState = {
    * @beta
    */
   onResolveDeepNoiseSuppressionDependency?: () => Promise<DeepNoiseSuppressionEffectDependency>;
-
+  /* @conditional-compile-remove(DNS) */
+  /**
+   * State to track whether the noise suppression should be on by default.
+   * @beta
+   * @default false
+   */
+  deepNoiseSuppressionOnByDefault?: boolean;
```

Comment:
I am guessing we are breaking the opt out default for this feature because of the lazy loading needed?

Reply:
Yes and also we want Contoso to be able to enable it if they need it by default. But it is a good point to discuss in feature weekly

Reply:
given the lazy loading I think its fine to have to opt in, this is the pattern for video effects too right?